gnucash/import-export/ofx/gnc-ofx-import.cpp

```diff
@@ -1294,28 +1269,28 @@ make_date_amount_key (time64 date, gnc_numeric amount)
 }
 
 static void
-runMatcher (ofx_info* info, const char* selected_filename,
+runMatcher (OfxInfo* info, const char* selected_filename,
             bool go_to_next_file)
 {
-    GtkWindow *parent = info->parent;
+    auto parent{static_cast<GtkWindow*>(info->parent)};
```

Comment:
It's generally wise to use the GObject casting macros--in this case `GTK_WINDOW`--followed by a null-check to confirm that it's a valid cast. This is equivalent to a C++ `dynamic_cast` that also requires a null-check.